flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -314,6 +316,10 @@ class AndroidNotificationDetails {
   /// Specifies the "ticker" text which is sent to accessibility services.
   final String? ticker;
 
+  final String? bubbleActivity;
```

Comment:
can you API docs like @Levi-Lesches mentioned? I also agree with the remark this should part of another class. even on the native Android side, these there's `BubbleMetadata` class so this could be used as the class' name too